install/kubernetes/helm/subcharts/security/templates/enable-mesh-mtls.yaml

```diff
@@ -23,7 +23,11 @@ apiVersion: networking.istio.io/v1alpha3
 kind: DestinationRule
 metadata:
   name: "default"
+  {{- if .Values.global.configRootNamespace }}
+  namespace: {{ .Values.global.configRootNamespace }}
```

Comment:
Actually `helm/tiller` is supposed to manage resources in one namespace, so when the `configRootNamespace` is different from `Release.Namespace`, there will be issue about chart upgrade/deleting.

Reply:
@diemtvu See https://github.com/helm/helm/issues/980#issuecomment-280167147

Reply:
@morvencao I parsed the issue tracker the same way as you did. Would you be willing to test if this is still the case in 2019?  (The issue was filed in 2017).  If this doesn't work well, I'm not sure what can be done about it, other then to create unmanaged objects for these types of things.

Reply:
@sdake @diemtvu I have done some local test in my env, the results are positive. Wait for response from helm guys , see: https://github.com/helm/helm/issues/980#issuecomment-462599157